src/client/components/sftp/file-item.jsx

```diff
@@ -645,7 +645,7 @@ export default class FileSection extends React.Component {
     window.pre.runGlobalAsync('watchFile', tempPath)
     fs.openFile(tempPath)
       .catch(window.store.onError)
-    window.pre.showItemInFolder(tempPath)
+    window.pre.openPath(tempPath)
```

Comment:
`fs.openFile(tempPath)` already opened file, just sometimes it may fail, so use `showItemInFolder` as a failsafe, can you explain why use `openPath`?
@iaoongin